.github/workflows/ci-actions.yml

```diff
@@ -42,6 +43,13 @@ jobs:
   build-jdk8:
     name: "JDK 8 Build"
     runs-on: ubuntu-latest
+    # Skip draft PRs and those with WIP in the subject, rerun as soon as its removed
+    if: "github.event_name != 'pull_request' || ( \
+           github.event.pull_request.draft == false && \
+           github.event.pull_request.state != 'closed' && \
+           contains(github.event.pull_request.title, 'wip ') == false && \
+           (github.event.action != 'edited' || contains(github.event.changes.title.from, 'wip ')) \
```

Comment:
We have also people using `[WIP]` so we should probably take it into account.

Also is `contains` case insensitive?

Reply:
good point. i can add that as well

Yes its insensitive